<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonSolverSetupPreferencePage</class>
 <widget class="QWidget" name="PythonSolverSetupPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Additional directories:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="directoryListWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="addNewDirectoryButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/CRIMSONSolverSetup.qrc">
           <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="openNewDirectoryButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/CRIMSONSolverSetup.qrc">
           <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeDirectoryButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/CRIMSONSolverSetup.qrc">
           <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>319</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/CRIMSONSolverSetup.qrc"/>
 </resources>
 <connections/>
</ui>
